src/CMakeLists.txt

```
@@ -234,6 +234,8 @@ install( FILES ${CMAKE_BINARY_DIR}/schema.xsd
 ################################
 # message(WARNING "Temporarily changing the geosPythonBranch to cusini/fix-streak-cert-fail")
 # set(GEOS_PYTHON_PACKAGES_BRANCH "cusini/fix-streak-cert-fail" CACHE STRING "" FORCE)
+message(WARNING "Temporarily changing the geosPythonBranch to feature/sherman/addPygeosxIntegratedTests")
+set(GEOS_PYTHON_PACKAGES_BRANCH "cusini/fix-streak-cert-fail" CACHE STRING "" FORCE)
```

you probably want to change this:
```suggestion
set(GEOS_PYTHON_PACKAGES_BRANCH "feature/sherman/addPygeosxIntegratedTests" CACHE STRING "" FORCE)
```